src/qt/transactionrecord.h

```diff
@@ -84,25 +84,34 @@ class TransactionRecord
         NameOp,
     };
 
+    enum NameOpType
```

Comment:
Unless there is a Qt-specific reason against it, I think you should use `enum class` here.  This makes `NameOpType` more type-safe.

Reply:
Good point.  I was originally trying to copy the surrounding style, but you're correct that the Bitcoin Core policy is that new code should use scoped enums.  Fixed now.